hashring/hashring_test.go

```diff
@@ -92,8 +120,25 @@ func TestServerCount(t *testing.T) {
 	assert.Equal(t, 2, ring.ServerCount(), "expected two servers to be in ring")
 }
 
+func TestCopyServers(t *testing.T) {
+	ring := New(farm.Fingerprint32, 10)
+	assert.Equal(t, 0, ring.ServerCount(), "expected one server to be in ring")
+
+	ring.AddServer("server1")
+	ring.AddServer("server2")
+	ring.AddServer("server3")
+
+	servers := ring.CopyServers()
```

Comment:
I think `Servers()` is a better name. The fact that they're being copied doesn't need to be in the name. It can just be in the documentation.